api/src/LinkyLink/Controllers/OpenIdController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace LinkyLink.Controllers
+{
+    /// <summary>
+    /// Manages endpoints for openid configuration pass through
```

Comment:
It would be nice to add a comment to describe why do we need this or link it to a backlog item/bug report if we have one?